### DISCUSSION TITLE
Is that correct that I don't have “homebrew/cask” tap after fresh brew instalation?

### DISCUSSION THREAD

**Discussion by sgon00** [Homebrew/discussions#1147] [Casks]
I just fresh installed MacOS Big Sur. 

And then I fresh installed `brew`.

During brew installation, there was one problem due to unstable network similar to https://github.com/Homebrew/brew/issues/10368. 
But after I re-ran it two more times after `rm -fr $(brew --repo homebrew/core)`, brew was installed successfully. (it said so).

But, I found out it didn't have `homebrew/cask` tap. Is that correct?

The reason why I think it might not be correct because I can not info any cask package.

```
% brew tap   
homebrew/core

% brew info firefox
Error: No available formula or cask with the name "firefox".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
```

**Comment by Rylan12** [comment]
> But, I found out it didn't have `homebrew/cask` tap. Is that correct?

Yes, homebrew/cask isn't tapped by default.

You can always manually run `brew tap homebrew/cask` to tap it.

You should also be able to run `brew install` on a cask and the repo will be automatically tapped. I thought we had that set-up for `brew info` too but I might be wrong (given your output).

Just to check, you're using a fresh install, right? Once you run `brew untap homebrew/cask` Homebrew will remember that you've untapped it and won't automatically re-tap. Does the following command have any output?

```sh
git -C $(brew --repository) config --get homebrew.untapped
```

**Reply by sgon00** [reply]
Thanks a lot for your reply. I haven't tried to brew install any casks yet. Because I want to verify `info` first before installing anything for security reasons.

And as what I said in the first post, it's a fresh install of `brew` and a fresh install of MacOS Big Sur too.

The output of the command you asked to run:

```
% git -C $(brew --repository) config --get homebrew.untapped
% echo $?
1
```

**Reply by Rylan12** [reply]
Okay, thanks. Everything looks good to me: no need to worry. I guess the auto-tap feature doesn't work for `brew info`.

Just run `brew tap homebrew/cask` and you'll be able to run `brew info` without any problems. `brew tap` won't install anything for you, it just downloads the cask definitions so that Homebrew can install them.

**Reply by sgon00** [reply]
Thank you very much for your answer. Now, I knew I should add the cask tap manually after fresh installation to use `info` command on cask formula.
